ddtrace/internal/remoteconfig/__init__.py

```diff
@@ -63,6 +63,13 @@ def register(cls, product, handler):
         except Exception:
             log.warning("error starting the RCM client", exc_info=True)
 
+    @classmethod
+    def unregister(cls, product):
+        try:
+            cls._worker._client.unregister_product(product)
```

Comment:
how does this relate to the IP blocking change?

would it be cleaner to introduce a PR just for registering/unregistering products with RCM?

Reply:
I will let @avara1986 comment on this but I think is needed since we register for the first time a RCM feature and thus have to get these functions.

Reply:
@brettlangdon we add this feature in this PR because IP blocking is the only Remote Configuration product that needs the unregister function